src/coremidi.cs

```diff
@@ -228,5 +253,172 @@ interface MidiCISession
 
 		[NullAllowed, Export ("propertyChangedCallback", ArgumentSemantic.Assign)]
 		MidiCIPropertyChangedHandler PropertyChangedCallback { get; set; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("initWithDiscoveredNode:dataReadyHandler:disconnectHandler:")]
+		IntPtr Constructor (MidiCIDiscoveredNode discoveredNode, Action handler, MidiCISessionDisconnectHandler disconnectHandler);
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("sendProfile:onChannel:profileData:")]
+		bool SendProfile (MidiCIProfile profile, byte channel, NSData profileSpecificData);
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("deviceInfo")]
+		MidiCIDeviceInfo DeviceInfo { get; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("maxSysExSize")]
+		NSNumber MaxSysExSize { get; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("maxPropertyRequests")]
+		NSNumber MaxPropertyRequests { get; }
+
+		[Internal]
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("midiDestination")]
+		MidiObjectRef _MidiDestination { get; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		MidiEndpoint MidiDestination {
+			[Wrap ("new MidiEndpoint (_MidiDestination)")]
```

Comment:
ouch, that returns a new instance each time the property is accessed :(
bad for a watch in the debugger (among other things)

Reply:
So, CoreMIDI is not present in the watch. But I am more worried of having a private copy of the MidiEndpoint, the issue is that the users might do something like calling the Dispose method on the property value and then have issues. 

If we do anything, I would move away from a property, do a GetMidiEndpoint or return the byte.